developer/src/common/web/utils/package.json

```diff
@@ -8,6 +8,9 @@
   "files": [
     "/build/"
   ],
+  "dependencies": {
+    "@sentry/node": "^7.57.0"
```

Comment:
consistent with version used in Keyman Developer Server; this was missing and should have been present